src/ServiceControlInstaller.Engine/Instances/ServiceControlCoreTransports.cs

```diff
@@ -10,93 +10,93 @@ public class ServiceControlCoreTransports
         {
             new TransportInfo
             {
-                Name = "AmazonSQS",
+                Name = TransportNames.AmazonSQS,
                 ZipName = "AmazonSQS",
                 TypeName = "ServiceControl.Transports.SQS.SQSTransportCustomization, ServiceControl.Transports.SQS",
                 SampleConnectionString = "AccessKeyId=<ACCESSKEYID>;SecretAccessKey=<SECRETACCESSKEY>;Region=<REGION>",
-                Matches = name => name.Equals("ServiceControl.Transports.SQS.SQSTransportCustomization, ServiceControl.Transports.SQS", StringComparison.OrdinalIgnoreCase)
-                                || name.Equals("AmazonSQS", StringComparison.OrdinalIgnoreCase)
+                Matches = name => name.Equals(TransportNames.AmazonSQS, StringComparison.OrdinalIgnoreCase) 
+                                  || name.Equals("ServiceControl.Transports.SQS.SQSTransportCustomization, ServiceControl.Transports.SQS", StringComparison.OrdinalIgnoreCase)
             },
             new TransportInfo
             {
-                Name = "AzureServiceBus - Endpoint-Oriented Topology",
+                Name = TransportNames.AzureServiceBusEndpointOrientedTopology,
                 TypeName = "ServiceControl.Transports.ASB.ASBEndpointTopologyTransportCustomization, ServiceControl.Transports.ASB",
                 ZipName = "AzureServiceBus",
                 SampleConnectionString = "Endpoint=sb://[namespace].servicebus.windows.net; SharedSecretIssuer=<owner>;SharedSecretValue=<someSecret>",
-                Matches = name => name.Equals("AzureServiceBus", StringComparison.OrdinalIgnoreCase)
+                Matches = name => name.Equals(TransportNames.AzureServiceBusEndpointOrientedTopology, StringComparison.OrdinalIgnoreCase) 
+                                  || name.Equals("AzureServiceBus", StringComparison.OrdinalIgnoreCase)
```

Comment:
Same question on this one.

Reply:
Actually, this is on `AzureServiceBusEndpointOrientedTopology` here, but `AzureServiceBusForwardingTopology` in the other file, so this inconsistency doesn't seem right.

Reply:
In terms of the current name see [this](https://github.com/Particular/ServiceControl/pull/1443#discussion_r229220558).

When it comes to topologies the current configuration is correct. The ASB was configured with Endpoint topology in SC and Forwarding in SC Monitroing.

Reply:
Ah, okay that explains the difference then!